File: 06.04.2022/bubble_sort.py
from operator import le


arr = [5,7,9,2,66,55,2,1]

def bubble_sort(arr):
    for i in range(len(arr) - 1):
        for j in range(len(arr)-1 -i):
            if arr[j] > arr[j + 1]:
                arr[j],arr[j+1] = arr[j+1],arr[j]
    return arr
print(bubble_sort(arr))

File: 06.04.2022/bubble_sort.js
let arr = [5,7,9,2,66,55,2,1];
let temp


function buuble_sort(arr) {
    for(let i=0; i < arr.length; i++) {
        for(let j=0; j < arr.length - i; j++) {
            if (arr[j] > arr[j+1]) {
                temp = arr[j]; 
                arr[j] = arr[j+1];
                arr[j+1] = temp;
            }
        }
    }
    return arr;
}

console.log(buuble_sort(arr));